Makefile

```diff
@@ -36,6 +33,14 @@ js:
 		echo "\n// @license-end"  >> $$js; \
 	done
 
+components: $(COMPONENTS_DIR)/*.vue
+	mkdir --parents $(BUILD)
+	rm -rf $(BUILD)/components
+	for component in $^; do \
+		echo $$component; \
+		npx vue-cli-service build --mode development --no-clean --dest $(BUILD)/components --inline-vue --target wc --name "ol-$$(basename $$component .vue)" "$$component"; \
```

Comment:
we discussed this may be problematic going forward

Reply:
Added comment